src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Automation/Peers/MenuItemAutomationPeer.cs

```diff
@@ -316,6 +316,16 @@ internal void RaiseExpandCollapseAutomationEvent(bool oldValue, bool newValue)
                 newValue ? ExpandCollapseState.Expanded : ExpandCollapseState.Collapsed);
         }
 
+        internal void RaiseToggleStatePropertyChangedEvent(bool oldValue, bool newValue)
```

Comment:
Add the NoInlining attribute, as in the previous method.

Reply:
Changes done and tested as per your suggestion.